cmd/notify-webhook.go

```diff
@@ -69,10 +69,15 @@ func isNetErrorIgnored(err error) bool {
 	}
 	switch err.(type) {
 	case net.Error:
-		switch err.(type) {
+		switch e := err.(type) {
 		case *net.DNSError, *net.OpError, net.UnknownNetworkError:
 			return true
 		case *url.Error:
+			// Fixes https://github.com/minio/minio/issues/4050
```

Comment:
Can you add a log when we ignore the error ? (in lookupEndpoint) it will be showed only one time and that would be better since we never try to send a notification again. (also since there is some code that will be removed in go 1.8 in lookupEndpoint, you don't need a perfect solution for the moment).